cli/test/lib/cli_spec.js

```diff
@@ -23,11 +23,11 @@ describe('cli', () => {
 
   beforeEach(() => {
     logger.reset()
-    sinon.stub(process, 'exit')
+    sinon.stub(process, 'exit').returns(null)
 
     os.platform.returns('darwin')
-    // sinon.stub(util, 'exit')
-    sinon.stub(util, 'logErrorExit1')
+    sinon.stub(util, 'logErrorExit1').returns(null)
```

Comment:
Correct errors in CI from test_setup:
<img width="999" alt="Screen Shot 2023-04-06 at 1 10 29 PM" src="https://user-images.githubusercontent.com/14099737/230461827-c4557415-7bad-4758-9517-2473ac6c593a.png">